src/main/java/VASSAL/counters/RestrictCommands.java

```diff
@@ -259,10 +259,11 @@ public String getState() {
     @Override
     public String getType() {
       SequenceEncoder se = new SequenceEncoder(';');
-      se.append(name.getValueString())
-      .append((actionOption.getSelectedIndex()==0) ? HIDE : DISABLE)
-      .append(propertyMatch.getValueString())
-      .append(watchKeys.getValueString());
+      se
```

Comment:
Why is this on it's own line now?

Reply:
Because it's only by chance that the length of the variable's name is equal to the indentation size, 2 characters. If the variable name changes, the indentation will stay properly indented.